scripts/sipify.py

```diff
@@ -3682,6 +3682,23 @@ def generate_python_output():
         )
         sys.exit(1)
 
+    # early handle of SIP_NO_FILE directive to avoid useless sip generation
```

Comment:
this is wrong -- we should be catching this earlier, in `sip_include.sh`, or in the pre-commit scripts themselves.